packages/common/src/adapters/trending.ts

```diff
@@ -1,27 +1,15 @@
 import { full } from '@audius/sdk'
 
-import { ID } from '~/models'
-import { TrendingIds } from '~/models/Trending'
-import { decodeHashId } from '~/utils'
+import { HashId } from '~/models'
 
-const makeIdList = (input: { id: string }[]): ID[] => {
-  return input.map(({ id }) => decodeHashId(id)).filter(Boolean) as ID[]
+const makeIdList = (input: { id: string }[]) => {
+  return input.map(({ id }) => HashId.parse(id))
```

Comment:
do we want to keep the filtering behavior? could to OptionalHashId.parse and filter nullable

Reply:
yeah good catch